sources/packages/backend/libs/services/src/health-check/health.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Global, Module } from "@nestjs/common";
+import { HealthService, ZeebeHealthIndicator } from ".";
+import { TerminusModule } from "@nestjs/terminus";
+
+@Global()
+@Module({
+  imports: [TerminusModule],
+  providers: [HealthService, ZeebeHealthIndicator],
+  exports: [TerminusModule, HealthService, ZeebeHealthIndicator],
```

Comment:
Do we need to export `TerminusModule` here?

Reply:
yes the compilation fails if we remove it.